.changeset/shiny-bulldogs-bathe.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.organization-discovery.v1": patch
+"@wso2is/i18n": patch
+---
```

Comment:
Let's add a separate changeset with `minor` update for server-configurations package too.